extensions/interactions/base_test.py

```diff
@@ -48,11 +48,6 @@
     ('show_generic_submit_button', bool)]
 
 
-def mock_get_filename_with_dimensions(filename, unused_exp_id):
```

Comment:
This function and its use in this test file is not needed

Reply:
This means the behavior changed at some point but we didn't update the tests?

Reply:
Yea that's correct